furms-persistence-spi/src/main/java/io/imunity/furms/spi/sites/SiteRepository.java

```diff
@@ -14,14 +14,18 @@
 
 	Optional<Site> findById(String id);
 
+	Optional<String> findShortId(String id);
```

Comment:
This name is very cryptic. What it does?
Also please introduce a type for short id instead of string.